src/Navigation/Xamarin.Forms/Navigation.Xamarin.Forms/Service/NavigationService.cache.cs

```diff
@@ -3,13 +3,13 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+
 using Xamarin.Forms;
 
 
 namespace CodeMonkeys.Navigation.Xamarin.Forms
 {
-    public partial class NavigationService :
-        INavigationService
+    public partial class NavigationService
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5598919)